### PR TITLE
Task 53 / PullApprove, Travis, Appveyor

### PR DESCRIPTION
**Pull request by ilia-glushchenko**

### CODE REVIEW

**Comment by ilia-glushchenko** [comment]
Approve

**Comment by ilia-glushchenko** [comment]
WIP

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    conditions:
+      branches:
+        - master  
+      files:
+        include:
+          - '*.md'
+          - '*.cpp'
+          - '*.c'
+          - '*.hpp'
+          - '*.h'
```
Comment (dsodintsov) [inline .pullapprove.yml]: `.cmake`
`CMakeLists.txt`

File: demo/Falling.cpp [hunk 2]
```diff
@@ -206,7 +188,7 @@ void FallingDemo::sceneReset()
         return distribution(generator);
     };
 
-    //Create particles
+    //Create m_particles
```
Comment (dsodintsov) [inline demo/Falling.cpp]: Can be left as `// Create particles`

Comment (dsodintsov) [inline demo/Falling.cpp]: Same for further comments that also have renamed variable

File: demo/Falling.cpp [hunk 3]
```diff
@@ -388,7 +370,7 @@ void FallingDemo::Display()
             double const r = sphere->GetRadius();
             glTranslatef(p.x, p.y, p.z);
 
-            if (&*activeObject != &body)
+            if (&*activeObject !=&body)
```
Comment (dsodintsov) [inline demo/Falling.cpp]: Can be reverted

File: demo/Falling.cpp [hunk 4]
```diff
@@ -368,7 +350,7 @@ void FallingDemo::Display()
             };
 
             glBegin(GL_QUADS);
-            if (&*activeObject != &body)
+            if (&*activeObject !=&body)
```
Comment (dsodintsov) [inline demo/Falling.cpp]: Can be reverted

File: demo/Falling.cpp [hunk 5]
```diff
@@ -401,7 +383,7 @@ void FallingDemo::Display()
                 glutSolidSphere(r, 20, 20);
             } 
 
-            if (&*activeObject != &body && !WIRED_ONLY) {
+            if (&*activeObject !=&body&& !WIRED_ONLY) {
```
Comment (dsodintsov) [inline demo/Falling.cpp]: revert

File: demo/Falling.cpp [hunk 6]
```diff
@@ -413,12 +395,12 @@ void FallingDemo::Display()
...
-            glm::dvec3 const & i = boxAxes[0];
-            glm::dvec3 const & j = boxAxes[1];
-            glm::dvec3 const & k = boxAxes[2];
+            glm::dvec3 const& i = boxAxes[0];
+            glm::dvec3 const& j = boxAxes[1];
+            glm::dvec3 const& k = boxAxes[2];
             std::array<glm::dvec3, 8> boxVertices = { i + j + k, i - j + k, -i + j + k, -i - j + k,
                                                       i + j - k, i - j - k, -i + j - k, -i - j - k };
-            if (&*activeObject != &body)
+            if (&*activeObject !=&body)
```
Comment (dsodintsov) [inline demo/Falling.cpp]: revert

File: demo/Falling.cpp [hunk 7]
```diff
@@ -468,7 +450,7 @@ void FallingDemo::Display()
             }
 
             //Draw wired Cube
-            if (&*activeObject != &body && !WIRED_ONLY) {
+            if (&*activeObject !=&body&& !WIRED_ONLY) {
```
Comment (dsodintsov) [inline demo/Falling.cpp]: revert

File: travisBuildScript.sh [hunk 8]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+mkdir $BUILD_DIR
+cd $BUILD_DIR
+
+CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=$COMPILER"
```
Comment (dsodintsov) [inline travisBuildScript.sh]: Do you need to also pass the `COMPILER_FLAGS` variable or does cmake fetch it from `env`?

File: demo/Falling.cpp (hunk 4 above)
Comment (ilia-glushchenko) [inline demo/Falling.cpp]: I will more careful next time doing replace all. Thanks.

File: travisBuildScript.sh (hunk 8 above)
Comment (ilia-glushchenko) [inline travisBuildScript.sh]: @dsodintsov I've faced some problems with the clang build on the Travis. After some googling, I end up with the solution needing to pass a flag to the compiler. I've figured it's not a CMake problem, but more of a correct clang setup. So currently I'm just setting it in the .travis.yml.

File: .pullapprove.yml (hunk 1 above)
Comment (ilia-glushchenko) [inline .pullapprove.yml]: Fixed

**Pull request merged**